jqjq.jq

```diff
@@ -1159,12 +1159,15 @@ def _tojson($opts):
   def _c_field: 7;
   def _color($id):
     if $opts.colors != null then
-      "\u001b[\($opts.colors[$id])m" + . + "\u001b[0m"
+      $opts.colors[$id], ., "\u001b[0m"
     else . end;
-  def _f($opts; $indent):
-    def _r($prefix):
+  (
+    ($opts.indent // 0) as $indent
```

Comment:
Some weird indent here?

Reply:
Oops. A bit of my style clashing with yours slipped in.

Reply:
Spotted another in `def dump`

Reply:
I'm a sucker for "haskell"-style :) think it works well with jq